apps/web/src/pages/Pools/PoolAdvertisementPage/components/DeadlineValue.tsx

```diff
@@ -0,0 +1,48 @@
+import { ReactNode } from "react";
+import { useIntl } from "react-intl";
+
+import { parseDateTimeUtc, formatDate } from "@gc-digital-talent/date-helpers";
+import { commonMessages } from "@gc-digital-talent/i18n";
+
+interface DeadlineValueProps {
+  closingDate: string | null | undefined;
+  closingReason: string | null | undefined;
+}
+
+const DeadlineValue = ({
+  closingDate,
+  closingReason,
+}: DeadlineValueProps): ReactNode => {
+  const intl = useIntl();
+  const notAvailable = intl.formatMessage(commonMessages.notAvailable);
+
+  if (closingDate && !closingReason) {
+    return intl.formatMessage(
+      {
+        defaultMessage: "Apply on or before {closingDate}",
+        id: "LjYzkS",
+        description: "Message to apply to the pool before deadline",
+      },
+      {
+        closingDate: formatDate({
+          date: parseDateTimeUtc(closingDate),
+          formatString: "PPP",
+          intl,
+          timeZone: "Canada/Pacific",
+        }),
+      },
+    );
+  }
+
+  if (closingReason) {
+    return intl.formatMessage({
+      defaultMessage: "<red>This advertisement has closed early.</red>",
```

Comment:
Contrast error on dark mode. Think we may need to update the base component to use a different colour.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/9863dfd4b65a1e62a564a9e13298f98a9e408f0f/packages/i18n/src/components/richTextElements.tsx#L64-L70

Reply:
I copied the colours of `heavyRed` in https://github.com/GCTC-NTGC/gc-digital-talent/pull/10991/commits/8f52e01cca9a2166192698884819605dffbd6f23